### DISCUSSION TITLE
運行程式表示已啟動，但是沒有顯示任何資訊

### DISCUSSION THREAD

**Discussion by mcakatsuki** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#23] [Q&A]
作者大大你好，在更新了最新檔後，運行程式表示已啟動，但是在馬娘中做任何操作都沒有資訊顯示。
使用的是umamusume-localify
CONFIG中已設定"notifier_host": "http://127.0.0.1:4693",
運行netstat命令也確定是監聽中的。
請問要怎麼辦呢?謝謝!

**Comment by EtherealAO** [comment]
https://github.com/EtherealAO/UmamusumeResponseAnalyzer/issues/4
你是不是和这位一样也把启用当禁用，把所有功能都关了

**Reply by mcakatsuki** [reply]
> #4 你是不是和这位一样也把启用当禁用，把所有功能都关了

為了防止這個錯誤，我已經反覆把那些選項開了又關，關了又開的嘗試過了(哭

**Reply by EtherealAO** [reply]
确认一下，umamusume-localify用的是不是原版的[umamusume-localify](https://github.com/GEEKiDoS/umamusume-localify)，原版是没有这个功能的。修改版https://github.com/EtherealAO/umamusume-localify 不需要设置notifier_host

**Reply by mcakatsuki** [reply]
> 确认一下，umamusume-localify用的是不是原版的[umamusume-localify](https://github.com/GEEKiDoS/umamusume-localify)，原版是没有这个功能的。修改版https://github.com/EtherealAO/umamusume-localify 不需要设置notifier_host

大大,抱歉原來是有這樣分的哦，原諒一下我是小白

我看到你回覆後我把修改版umamusume-localify的檔案放了進去替換，可是替換後遊戲是可以正常運行，但當想運行UmamusumeResponseAnalyzer的時候卻出現和隔壁一樣出現閃退的問題，用的也是你剛發表的1.8.7的檔案。

**Reply by mcakatsuki** [reply]
補充一下...我把主頁的那些.JSON檔案放到LOCAL APP DATA的檔案後，Analyzer可以起動了，可是仍舊是無法在遊戲的操作中得到資訊的顯示。

**Reply by mcakatsuki** [reply]
不知道為什麼...過了一晚後它又可以正常運作了...UN.....總之感謝大大
